spanner/api/Spanner.Samples/EnableFineGrainedAccess.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_enable_fine_grained_access]
+
+using Google.Api.Gax;
+using Google.Cloud.Iam.V1;
+using Google.Cloud.Spanner.Admin.Database.V1;
+
+public class EnableFineGrainedAccessSample
+{
+    public Policy EnableFineGrainedAccess(string projectId, string instanceId, string databaseId, string databaseRole, string iamMember)

```

Comment:
When these samples are shown in docs, it is best if users don't have to scroll to the right. We don't have a hard line length rule, but we try to avoid really long lines in samples.
Please check the others.

```suggestion
    public Policy EnableFineGrainedAccess(
    string projectId, string instanceId, string databaseId,
    string databaseRole, string iamMember)

```